common/json/impl/boost/boost_json.cpp

```diff
@@ -0,0 +1,21 @@
+#include <iostream>
+#include <common/json/impl/boost/boost_json.hpp>
+#include <boost/json/src.hpp>
```

Comment:
I would prefer us not using header-only versions of the libraries and just statically-link the libraries with our code. Would that be a major obstacle/complication?

Reply:
Boost JSON seems to be a an exception, in that provides both static library and single header file approach. For the other Boost libraries that we have looked into they either provide one or the other.

I _think_ that the end result is the same, and with static library approach the compile time reduces greatly during local development (like you build it once, while with the header you will need to compile the objects every time you make a change in the client code, `bost_json.cpp` in this case).

No major obstacle/complication, I just wanted to keep my prototype simple :+1: 

So for json please go ahead and rework this to use the static library. Just keep in mind that for other libraries we'll go for the header file approach because they don't provide the static library approach.